cclass.py

```diff
@@ -72,7 +72,7 @@ def parseClass(m, compendium, args):
     SFText.text = "• Weapons: " + weapontext
     SFText = ET.SubElement(StartingFeature, 'text')
     if "tools" in m['startingProficiencies']:
-        SFText.text = "• Tools: " + ", ".join(m['startingProficiencies']['tools'])
+        SFText.text = str(utils.fixTags("• Tools: " + ", ".join(m['startingProficiencies']['tools']),m,args.nohtml))
```

Comment:
The str() cast is unnecessary as utils.fixTags() returns a String

Reply:
Removed str() casts.